src/Splitio-net-core/Services/Cache/Classes/InMemoryReadinessGatesCache.cs

```diff
@@ -36,11 +36,11 @@ public void SplitsAreReady()
                 splitsAreReady.Signal();
                 if (splitsAreReady.IsSet)
                 {
-                    Log.Info("Splits are ready");
+                    Log.Debug("Splits are ready");
```

Comment:
is it possible to modify this to: `Split are ready in xxx millis` ?

Reply:
Sure, I can do some changes to the class in order to log the time it took. I can start a 2 timers when the class gets constructed and then check them for logging